### ISSUE TITLE
Editing a transaction amount defaults to expense

### ISSUE THREAD

**Issue by torrentails** [asafonov/monly.apk#7]
When editing a transaction's amount, the transaction type always switches to `expense`, even if it was set as `income` initially.

**Comment by torrentails** [comment]
I think it's because of this line, [/app/src/main/assets/main.js#L297](https://github.com/asafonov/monly.apk/blob/fb56f5f7a22c5c88dd5abf77cbc214358f2ef9f9/app/src/main/assets/main.js#L297), which sets the value based on positivity of the amount

However, I tried testing it and even entering a negative number still sets the transaction as an `expense`

**Comment by asafonov** [comment]
yes, the amount is always positive in user interaction, i just have to check the existing type of the transaction and adjusting the amount based on that. I will fix it, thank you
